sphinx-doc/testing.rst

```diff
@@ -19,12 +19,12 @@ APIs to provide inputs and check for correct outputs. For example, test that the
 hard sphere HPMC simulation executes for several steps. System integration tests
 may take several seconds.
 
-**Validation tests** rigorously check that HOOMD simulations sample the correct
-statistical ensembles. For example, validate the a Lennard-Jones simulation at a
-given density matches the pressure in the NIST reference. Validation tests
-should run long enough to ensure reasonable sampling, but not too long. These
-test run in a CI environment on every pull request. Individual validation tests
-should execute in less than 10 minutes.
+**Long running tests** check for correct behavior, but require up to a minute to execute. Mark long
+running tests with the ``validate`` label.
```

Comment:
I think it makes sense to keep a label for the longer running tests, but what if we changed the label to be `--long-tests` or something along those lines intead of `--validate`?

Reply:
This is a breaking change. We could do this in separate pull request for v4. The canonical name would be `slow`.

Reply:
Sounds good.